test/SILOptimizer/accessutils.sil

```diff
@@ -0,0 +1,324 @@
+// RUN: %target-sil-opt %s -dump-access | %FileCheck %s
```

Comment:
You need to add `-o /dev/null`. Otherwise the passe's print output and the output SIL get mixed up.
(this is the failure on the linux PR test)